plugin/src/sbt-test/sbt-avro/publishing/transitive/src/test/resources/test.avsc

```diff
@@ -6,10 +6,6 @@
     {
       "name": "stringField",
       "type": "string"
-    },
-    {
-      "name": "referencedTypeField",
-      "type": "com.github.sbt.avro.test.external.Avsc"
```

Comment:
To support such things, we would need to re-compile the schemas from the compile scope

Reply:
So in this case an external schemas lib can have no deps on other schema libs?

Reply:
we can (it is done in the `Compile` scope).
here we want the schemas in the `test` scope to depend on some other schema from the `compile`. To do so, we need to parse those again.